fmriprep/workflows/bold/base.py

```diff
@@ -564,18 +545,18 @@ def init_func_preproc_wf(bold_file, has_fieldmap=False):
     # MAIN WORKFLOW STRUCTURE #######################################################
     # fmt:off
     workflow.connect([
+        # Prepare masked T1w image
         (inputnode, t1w_brain, [("t1w_preproc", "in_file"),
                                 ("t1w_mask", "in_mask")]),
+        # Select validated bold files per-echo
+        (initial_boldref_wf, validated_bold, [("val_bold.out_file", "inlist")]),
```

Comment:
This violates our usual rules, so I can create a new `outputnode` entry in niworkflows, assuming this PR is approved in principle.

Reply:
+1 to adding a new outputnode field.